x/authz/spec/01_concepts.md

```diff
@@ -0,0 +1,37 @@
+<!--
+order: 1
+-->
+
+# Concepts
+
+## Authorization
+Any concrete type of authorization defined in the `x/authz` module must fulfill `Authorization` interface outlined below. Authorizations determine exactly what privileges are granted. They are extensible and can be defined for any Msg service method even outside of the module where the Msg method is defined. Authorizations use the new `ServiceMsg` type from [ADR 031](docs/architecture/adr-31-msg-service.md).
+
+
++++ https://github.com/cosmos/cosmos-sdk/blob/master/x/authz/types/authorizations.go#L15-L24
```

Comment:
let's use a commit hash instead of master, as master's code can change (Same applies to all other code snippets)